SonarQube.Common/AnalysisProperties/FilePropertyProvider.cs

```diff
@@ -75,6 +77,8 @@ public static bool TryCreateProvider(IEnumerable<ArgumentInstance> commandLineAr
 
         public AnalysisProperties PropertiesFile {  get { return this.propertiesFile; } }
 
+        public bool IsDefaultSettingsFile { get { return this.isDefaultPropertiesFile; } }
```

Comment:
Modified the file provider to say whether it is using the default properties file or not.